bundles/org.openhab.binding.airq/README.md

```diff
@@ -0,0 +1,224 @@
+img[src*="#small"] {
+    width:400px;
+    height:324px;
+}
+
+# air-Q Binding
+
+The air-Q Binding integrates the air analyzer [air-Q](http://www.air-q.com) device into the openHAB system.
+
+With the binding, it is possible to subscribe to all data delivered by the air-Q device.
+
+![air-Q image](doc/image_air-Q.png#small)
+
+## Supported Things
+
+Only one Thing is supported: The `air-Q` device.
```

Comment:
Better use the Thing Type ID here.
```suggestion
Only one Thing is supported: The `airq` device.
```

Reply:
Actually I tried to use airq when it was about the binding (because the binding name does not allow the hyphen), and air-Q when it is about the device. OK if I leave it like this?

Reply:
I meant only this occurrence as users who use textual configuration need it, if that answers you question.

Reply:
Can you comment on this?